lib/src/ui/home_page.dart

```diff
@@ -36,9 +37,57 @@ class HomePageState extends State<HomePage> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(
-        title: Text(
-          widget.title,
+      appBar: PreferredSize(
+        preferredSize: Size.fromHeight(
+          Constants.preferredSizeFromHeight,
+        ),
+        child: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Padding(
+                padding: EdgeInsets.all(
+                  Constants.childrenPadding,
+                ),
+                child: Text(
+                  widget.title,
+                  textAlign: TextAlign.center,
+                  style: TextStyles.styleTitle,
+                ),
+              ),
+              Padding(
+                padding: EdgeInsets.all(
+                  Constants.childrenPadding,
+                ),
+                child: SizedBox(
+                  child: TextField(
+                    style: TextStyles.styleSearch,
+                    decoration: InputDecoration(
+                      prefixIcon: Padding(
+                        padding: EdgeInsets.only(
+                          right: Constants.paddingIcon,
+                        ),
+                        child: Icon(
+                          Icons.east,
+                          color: Colors.black,
+                        ),
+                      ),
+                      enabledBorder: UnderlineInputBorder(
+                        borderSide: BorderSide(
+                          color: Colors.black,
+                        ),
+                      ),
+                    ),
+                    onSubmitted: (nameMovie) {
+                      widget.bloc.searchByMovieName(nameMovie);
+                    },
+                  ),
+                  width: Constants.widthSearch,
+                  height: Constants.heightSearch,
```

Comment:
You can change this with a more descriptive constant name like searchSixedBoxWidth and searchSixedBoxHeight

Reply:
fixed